RecoLocalCalo/HGCalRecProducers/interface/HGCalUncalibRecHitWorkerBaseClass.h

```diff
@@ -0,0 +1,24 @@
+#ifndef RecoLocalCalo_HGCalRecProducers_HGCalUncalibRecHitWorkerBaseClass_hh
+#define RecoLocalCalo_HGCalRecProducers_HGCalUncalibRecHitWorkerBaseClass_hh
+
+#include "DataFormats/HGCRecHit/interface/HGCRecHitCollections.h"
+#include "DataFormats/HGCDigi/interface/HGCDigiCollections.h"
+
+namespace edm {
+        class Event;
+        class EventSetup;
+        class ParameterSet;
+}
+
+class HGCalUncalibRecHitWorkerBaseClass {
+        public:
+                HGCalUncalibRecHitWorkerBaseClass(const edm::ParameterSet&){}
+                virtual ~HGCalUncalibRecHitWorkerBaseClass(){}
+
+                virtual void set(const edm::EventSetup& es) = 0;
```

Comment:
some comments on the purpose of the methods would be nice
(not a showstopper for the PR)